images/common/init_command.sh

```
@@ -30,16 +30,13 @@ elif [ "$MODULE_NAME" = 'freeradius' ]; then
 	fi
 elif [ "$MODULE_NAME" = 'openvpn' ]; then
 	if [[ -z "$VPN_DOMAIN" ]]; then exit; fi
+	source openvpn_utils.sh
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'source' in place of '.' is undefined.

(SC3046)